docs/en/source/introduction.rst

```diff
@@ -5,7 +5,8 @@ Introduction
 Scorpio broker is a reference implementation of **NGSI-LD standard** specifications that are compliant to **ETSI standards**. Basically Scorpio broker is a core component of **FiWARE/IoT** platform wherein IoT data-driven by dynamic context is collected, processed, notified & stored/ingested with different application usage perspectives.
 Scorpio broker also provides an implementation of REST API endpoints for various data context operations that conform to **NGSI-LD API** specification.
 Scorpio broker allows you to collect, process, notify and store the IoT data with dynamic context with the use of linked data concepts.
-It makes use of the **microservice-based architecture** build with the help of **spring boot**, which has its own advantages over the existing IoT brokers such as scalability, cross-technology integration, etc.
+
+Scorpio Broker makes use of the **microservice-based architecture**. Scorpio Broker currently is provided in two versions. One build with **Spring Boot** and new main version build with **Quarkus**, which has its own advantages over the existing IoT brokers such as scalability, cross-technology integration, etc.
```

Comment:
Remove this line "Scorpio Broker currently is provided in two versions. One build with **Spring Boot**"

Reply:
Done